src/App.js

```diff
@@ -25,7 +29,24 @@ class App extends Component {
                 { name: 'Jared', id: 0 },
                 { name: 'Adrian', id: 1 }
             ],
+            addingPlayer: false
         }
+        this.id = 1
```

Comment:
Having data outside of the state might end up with some unintended consequences. I don't know what best practices are for unique id's in react might be worth a search.